Added Profiling elastic agent integration (#2996)
elastic/observability-docs: docs/en/observability/profiling-get-started.asciidoc

[discrete]
[[profiling-install-host-agent]]

> == Install the host-agent in standalone mode

Reviewer: I think it's probably a good idea to tell people that there are two options before this section so they don't start with standalone and realize they want to install with the agent. I tried to do this in suggestions, but wasn't sure I was getting the formatting right, so I'll give it a try here:

```
[discrete]
[[profiling-install-host-agent]]
== Install the host-agent

You have the following options when installing the host-agent:

- <<profiling-install-host-agent-standalone>>
- <<profiling-install-host-agent-elastic-agent>>

[discrete]
[[profiling-install-host-agent-standalone]]
=== Install the host-agent in standalone mode
```

Author: done

[role="screenshot"]
image::images/profiling-k8s-hostagent.png[]

[discrete]
[[profiling-install-host-agent-elastic-agent]]
== Install the host-agent via Elastic Agent 

> If you do not wish to install the host agent in standalone mode as shown in <<profiling-install-host-agent>>, you can instead allow Elastic Agent to orchestrate the installation by using the Universal Profiling Agent integration.

Reviewer: ```suggestion
You can also use the Universal Profiling Agent integration through the {agent} to orchestrate to install the host-agent.
```

To install the Universal Profiling Agent integration, complete the following steps:

. Copy the `secret token` and `APM Server URL` from the Elastic Agent Integration 
+
[role="screenshot"]
image::images/profiling-elastic-agent.png[]
+

> . Then click on step 2 to complete the integration. 

Author: @mdbirnstiehl  - the screenshot is included - https://github.com/elastic/observability-docs/blob/profiling-elastic-agent/docs/en/observability/profiling-get-started.asciidoc

> . Then click on step 2 to complete the integration. 

Reviewer: So do we want them to click here:

![image](https://github.com/elastic/observability-docs/assets/114418652/9a6d13d2-7d12-438b-a9e8-6aa8941ff066)

if so, I would say "Click **Manage Universal Profiling Agent in Fleet**."

> . Then click on step 2 to complete the integration. 

Author: done
